mod-calendar/src/main/resources/bama/mod-calendar/features/calendarPeriods.feature

```diff
@@ -82,8 +89,10 @@ Feature: Calendar periods
     Given path 'calendar', 'periods'
     When method GET
     Then status 200
-    And match $.openingPeriods[0].openingDay.open == true
```

Comment:
Same issue as [:44](https://github.com/folio-org/folio-integration-tests/pull/495/files#diff-58b96cd93d0a7cb94e0cbd76f2e5e4b2458bd1de94cc362bb487158ba3a385d5R44) (but on the starting end).